packages/formstr-app/src/containers/FormFillerNew/FormRendererContainer.tsx

```diff
@@ -29,6 +28,9 @@ export const FormRendererContainer: React.FC<FormRendererContainerProps> = ({
   const { Text } = Typography;
   const [formTemplate, setFormTemplate] = useState<Tag[]>();
   const [settings, setSettings] = useState<IFormSettings>();
+  const [searchParams] = useSearchParams();
+  const hideTitleImage = searchParams.get("hideTitleImage") === "true";
```

Comment:
should the searchParams not be used in `FormFillerNew/index.tsx` ? and passed here as props? why are we using search params here?